bundles/org.openhab.binding.haywardomnilogic/src/main/resources/OH-INF/thing/backyard.xml

```diff
@@ -33,7 +33,7 @@
 		<item-type>Number:Temperature</item-type>
 		<label>Air Temp</label>
 		<description>Air Temp</description>
-		<state pattern="%1f %unit%" readOnly="true"/>
+		<state pattern="%1.0f%unit%" readOnly="true"/>
```

Comment:
```suggestion
		<state pattern="%.1f%unit%" readOnly="true"/>
```

Reply:
The Omni Logix server only reports integer values for temperature.  Do you still prefer %.1f %unit% ?

Reply:
Also, any hints on how to handle the failed build on the org.openhab.persistence.dynamodb.internal.TestStoreMixedTypesTest ?

Reply:
I thought it was a typo. Integer would be `%d %unit`. The failed test seems unreleated to your changes.